Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/templates/azure-policy-serviceaccount.yaml

```diff
@@ -15,6 +15,9 @@ rules:
 - apiGroups: ["templates.gatekeeper.sh"]
   resources: ["constrainttemplates"]
   verbs: ["create", "delete", "update", "list", "get"]
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get"]
```

Comment:
Does this mean Azure-policy has permissions to get info on all pods? Would customer be fine with giving that level of information? Can we just restrict to a namespace or a particular pod?

Reply:
I've updated so the azure-policy only has permissions to get info on pods in the kube-system namespace. Unfortunately we can't restrict it any tighter than that.